core/src/components/textarea/textarea.ionic.scss

```diff
@@ -0,0 +1,80 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.common";
+@import "./textarea.ionic.vars";
+@import "./textarea.ionic.outline.scss";
```

Comment:
We should be using `@use` because:
1. That's how we've been setting it up so we would stay consistent with other `ionic` theme files
2. We would have less files to convert once we move from `@import`

Reply:
Good call, I copied this over from the `input` file which is an older implementation. Fixed in https://github.com/ionic-team/ionic-framework/pull/29776/commits/ea98cf34619813eebbd090272c124cdb515ba27b